src/components/table-of-contents/index.njk

```diff
@@ -8,22 +8,51 @@ title: Table of contents
 {% from "components/panel/_macro.njk" import onsPanel %}
 +++
 
-{{
-    onsPanel({
-        "body": 'This component requires documentation.'
-    })
-}}
+The table of contents lets users navigate between pages of a guide or jump to sections on a page.
+
 
-# Table of contents - simple
 {{
     patternlibExample({"path": "components/table-of-contents/examples/table-of-contents-single/index.njk"})
 }}
 
-# Table of contents - multiple sections
+## When to use this component
+Use this component at the top of a long page of content broken up by subheadings, or at the top of each page of a [guide](/patterns/guide).
+
+If you're using a table of contents, you should also use the [pager component](/components/pagination/#pager) at the bottom of the page. The 2 components make up the [guide pattern](/patterns/guide/).
+## When not to use this component
+Do not use the [in-page variant](#table-of-contents---in---page-navigation) of this component if the height the table of contents will be greater than the height of the browser‘s viewport. The list will not scroll independently of the page content because research shows some users find this confusing. 
```

Comment:
```suggestion
Do not use the [in-page variant](#table-of-contents---in-page-navigation) of this component if the height the table of contents will be greater than the height of the browser‘s viewport. The list will not scroll independently of the page content because research shows some users find this confusing. 
```